eclair-core/src/main/scala/fr/acinq/eclair/io/Peer.scala

```diff
@@ -191,6 +202,14 @@ class Peer(val nodeParams: NodeParams, remoteNodeId: PublicKey, wallet: OnChainA
             stay()
         }
 
+      case Event(SpawnChannelNonInitiator(msg, channelConfig, channelType, localParams), d: ConnectedData) =>
```

Comment:
Nit:

```suggestion
      case Event(SpawnChannelNonInitiator(open, channelConfig, channelType, localParams), d: ConnectedData) =>
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/2284/commits/5b956cb456de1ac54175013116985654f06622f9